packages/styles/src/variables/components/_badge.scss

```diff
@@ -1,19 +1,30 @@
-@use 'sass:math';
-
-@use './button';
-@use './../animation';
 @use './../color';
 @use './../commons';
 @use './../spacing';
 @use './../type';
 
+@use './button';
 @use './../../functions/sizing';
 
-// Design System custom variables
-$badge-height: button.$btn-height-rg;
+$badge-border-radius: 50rem;
+$badge-line-height: type.$line-height-copy;
+$badge-color: color.$white;
+$badge-bg: color.$error-red;
+$badge-border: color.$white solid commons.$border-thick;
```

Comment:
We decided to always use a white border because the background could also have an alpha.

Here with --post-bg-rgb: $dark and --post-bg-opacity: 0.5
![image](https://github.com/swisspost/design-system/assets/33580481/b0d2b791-35d0-46d3-a005-c6a9af1f7e3c)

Reply:
Yes, this would be different, when we would not use the background utilities, but instead add some badge-bg classes to color it. But whatever, it's just fine for me at the moment ;)